packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -539,10 +548,44 @@ function LineItemSublabelAndPrice( {
 	}
 
 	if (
-		( isDomainRegistration || isDomainMap || isGSuite ) &&
-		product.months_per_bill_period === 12
+		isGoogleWorkspaceProductSlug( productSlug ) ||
+		isGSuiteOrExtraLicenseProductSlug( productSlug ) ||
+		isTitanMail( product )
 	) {
+		let interval = null;
+
+		if ( product.months_per_bill_period === 12 || product.months_per_bill_period === null ) {
+			interval = translate( 'billed annually' );
+		}
+
+		if ( product.months_per_bill_period === 1 ) {
+			interval = translate( 'billed monthly' );
+		}
+
+		if ( interval === null ) {
+			return sublabel;
+		}
+
+		return (
+			<>
+				{ translate( '%(productType)s: %(interval)s', {
```

Comment:
Nit: the sublabel is not really a "type" but is more of a description. I also think it's worth being more explicit about the billing internal. So I would lean towards using something more like `'%(productDescription)s: %(billingInterval)s'` to make it even more explicit as to what each element is.

Reply:
I'll do that 👍.